corona-warn-app/cwa-app-android
# PPA onboarding screen (EXPOSUREAPP-4815)

Onboarding screen for PPA.

![Vuzix View_20210205_174918_174](https://user-images.githubusercontent.com/64849422/107048878-7c3f8080-67da-11eb-9e3b-4c50de819d91.jpg)

How to test:
1) Fresh install/Reset app
2) Check the last onboarding screen
3) Turn on Screen reader accessibility feature and play around the screen
